### PR TITLE
bsc tokens

### PR DESCRIPTION
**Pull request by akuter-bc**

### CODE REVIEW

File: chain/binance/tokens.json [hunk 1]
```diff
@@ -2636,6 +2744,15 @@
         "symbol": "AIDI.BNB",
         "website": "https://aidiverse.com"
     },
+    {
+        "address": "0xb3c11196A4f3b1da7c23d9FB0A3dDE9c6340934F",
```
Comment (akuter-bc) [inline chain/binance/tokens.json:2748]: again, there is another address: 0xb7F8Cd00C5A06c0537E2aBfF0b58033d02e5E094
But this is the new one apparently, see [here](https://paxos.com/2021/08/24/the-digital-dollar-that-always-equals-a-dollar-paxos-standard-pax-is-now-pax-dollar-usdp/)

Comment (akuter-bc) [reply chain/binance/tokens.json]: unsupported by coingecko, removed

**Review by pavel-bc — APPROVED** [review]
✅  Verified all tokens on [BscScan.com](https://bscscan.com/)

**Comment by jhuntbach-bc** [comment]
✅ Verified all tokens on CoinGecko

**Review by jhuntbach-bc — APPROVED** [review]

**Pull request merged**